privaterelay/settings.py

```diff
@@ -109,6 +109,7 @@
 GOOGLE_ANALYTICS_ID = config('GOOGLE_ANALYTICS_ID', None)
 RECRUITMENT_BANNER_LINK = config('RECRUITMENT_BANNER_LINK', None)
 RECRUITMENT_BANNER_TEXT = config('RECRUITMENT_BANNER_TEXT', None)
```

Comment:
```suggestion
RECRUITMENT_BANNER_TEXT = config('RECRUITMENT_BANNER_TEXT', None)
RECRUITMENT_EMAIL_BANNER_TEXT = config('RECRUITMENT_EMAIL_BANNER_TEXT', None)
```
I assume the `RECRUITMENT_BANNER_TEXT` config was for banner on the dashboard and we probably want a separate banner text like you did for the link?